foreshadow/cleaners/internals/__init__.py

```diff
@@ -33,5 +33,5 @@ def _get_classes():
     return classes
 
 
-classes = _get_modules(_get_classes(), globals(), __name__)
+classes = _get_modules(_get_classes(), globals(), __name__, wrap=False)
```

Comment:
Can we change this to concretely import the appropriate internals? That will help us when refactoring, so we can focus on changing the imports and not changing code implementation.

Reply:
Sorry? I don't follow.

Reply:
Will fix in my branch 'discussion around the usage of _get_modules' to only be a wrapper and stop the auto-importing from V1.